DS-Session4-Question1/src/meta/utility/SortByName.java

```diff
@@ -0,0 +1,31 @@
+package meta.utility;
+
+import java.util.Comparator;
+
+public class SortByName implements Comparable<SortByName>{
```

Comment:
I think, this class shall contain only compare method.
Take reference from: https://www.geeksforgeeks.org/collections-sort-java-examples/